superset/assets/src/dashboard/containers/Dashboard.jsx

```diff
@@ -49,18 +48,15 @@ function mapStateToProps(state) {
     dashboardState,
     charts,
     datasources,
-    // filters prop: All the filter_box's state in this dashboard
+    // filters prop:
+    // All the active filter_box's values and scope in this dashboard, for each filter field.
     // When dashboard is first loaded into browser,
     // its value is from preselect_filters that dashboard owner saved in dashboard's meta data
     // When user start interacting with dashboard, it will be user picked values from all filter_box
     filters: getActiveFilters(),
     slices: sliceEntities.slices,
     layout: dashboardLayout.present,
     impressionId,
-    loadStats: getLoadStatsPerTopLevelComponent({
```

Comment:
why are we removing this?

Reply:
this is old dashboard perf logging. this prop `loadStats` is not used anywhere.